### DISCUSSION TITLE
Hosting?

### DISCUSSION THREAD

**Discussion by denisgurev** [pocketbase/pocketbase#432] [Q&A] [answered]
I really like working with PocketBase locally but it's missing a public hosting like in Supabase.

Do you plan supporting that? I think a lot of newbies like me will gladly pay to not maintain servers.

**Comment by ganigeorgiev** [comment, marked as answer]
I've received several similar requests on the support mail, but for now I'm not planning to offer cloud hosting.

PocketBase is a hobby project and I don't think I'll ever have the time, or the resources, to manage a full-fledged cloud hosting.

If you are looking for an afforable hosting, take a look at [fly.io](https://fly.io/). I haven't been able to test them yet, but a lot of great people works there and they also have free plans with persistent storage included.

I'll try to find time after the users refactoring and will create a tutorial/guide how to deploy on fly.io.

Another hosting option that I could recommend, especially if you are from Europe, is [Hetzner](https://www.hetzner.com/cloud). PocketBase is completely portable, so you can spin a brand new Ubuntu VPS, upload the binary and just execute:

```
./pocketbase serve --http="yourdomain.com:80" --https="yourdomain.com:443"
```
(_an optional systemd service may be a good idea also to keep the process running in case of a restart; for more info check - https://pocketbase.io/docs/going-to-production/_)

**Reply by denisgurev** [reply]
Thanks for the quick reply! I'll give hetzner a try.

**Reply by miguelgargallo** [reply]
Now if you combine this with svelte --port

Answer: npm run dev --host network: not exposed https://stackoverflow.com/a/71941435/18185259?stw=2 

you can deploy servers with svelte and pocketbase at the same time!

thanks both of you

**Comment by benallfree** [comment]
@ganigeorgiev I've been playing around a bit with creating a multitenant pocketbase solution on fly.io. Is that something you'd be interested in chatting more about?

**Reply by ganigeorgiev** [reply]
> Is that something you'd be interested in chatting more about?

If you need help, feel free to post your question here or open a new discussion and I'll try to help, but keep in mind that #376 is with the highest priority right now and new features and PRs are put aside until v0.8.0.

**Comment by papakay** [comment]
I think this might be useful for someone - 

https://pockethost.io
https://elest.io/open-source/pocketbase

**Reply by benallfree** [reply]
PocketHost author here, thanks for the mention :)

PocketHost is (to my knowledge) the only open source hosting platform for PocketBase.

**Reply by miguelgargallo** [reply]
Wait! With coolify I host all my pocketbases open source at home for free!
